trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require './correct_nameable'
+class TrimmerDecorator < CorrectNameable
+  def correct_name
+    @person.correct_name.strip
+  end
+end
```

Comment:
- [ ] Kindly make sure that the app works as intended. Currently, if I were to input the following inputs:

![image](https://user-images.githubusercontent.com/62290545/225036054-95fd434e-8352-47a0-977d-e8591baa9ada.png)

- This is the result:

![image](https://user-images.githubusercontent.com/62290545/225036446-6cb4871d-2c3e-4736-9851-cbe1fa23a60e.png)

- The `CapitalizeDecorator` Class is supposed to capitalize the first letter of the string, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=For%20the%20CapitalizeDecorator,nameable.correct_name.) requirement.
- The `TrimmerDecorator` Class is supposed to make sure that the output has a maximum of 10 characters, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=For%20the%20TrimmerDecorator,trim%20the%20word.) requirement.